views/layouts/main.php

```diff
@@ -143,6 +143,18 @@
 //                            ]
                             //\SILEX:info
                         ]];
```

Comment:
Only the admin can have the Data Analysis tab in the menu. Is this normal?

Reply:
no

Reply:
Corrected .